pgdrive/world/chase_camera.py

```diff
@@ -161,9 +160,11 @@ def destroy(self, pg_world):
         if pg_world.taskMgr.hasTaskNamed(self.TOP_DOWN_TASK_NAME):
             pg_world.taskMgr.remove(self.TOP_DOWN_TASK_NAME)
 
-    def stop_chase(self, pg_world: PGWorld):
+    def stop_track(self, pg_world: PGWorld, current_chase_vehicle):
```

Comment:
we should also call this function in env.close()

Reply:
It is already called now